cabby/rvs/templates.py

```diff
@@ -307,12 +317,13 @@ def add_features_to_template(template: Text, entity: geo_item.GeoEntity) -> Tupl
   template = template.replace(' a e', ' an e')
   template = template.replace(' a u', ' an u')
   template = template.replace(' a o', ' an o')
```

Comment:
I realize it isn't part of this PR, but a nicer solution to this would be to have a regex like:

re_indef_vowel = re.compile(r'\ba ([aeiou])')

Then you just need:

template = re_indef_vowel.sub(r'an \1', template)

Reply:
Changed as suggested.